tests/testthat/helpers.R

```diff
@@ -11,7 +12,7 @@ create_minimal_package <- function(dir = tempfile()) {
 
   ## DESCRIPTION
   desc::description$new("!new")$
-    set(Package = basename(dir))$
+    set(Package = packagename)$
```

Comment:
the package name is needed otherwise the random name means the cached requests never correspond to the current requests.